flashinfer/fused_moe/core.py

```diff
@@ -236,7 +236,7 @@ def gen_cutlass_fused_moe_sm100_module(use_fast_build: bool = False) -> JitSpec:
         raise RuntimeError(f"Failed to generate Cutlass kernels: {e}") from e
 
     return gen_jit_spec(
-        "fused_moe_sm100",
+        "fused_moe_cutlass_sm100",
```

Comment:
Yes we should never collide for any two extensions, can you also rename the other module as `fused_moe_trtllm_sm100`?